WSLGd/main.cpp

```diff
@@ -388,6 +388,8 @@ try {
         std::vector<cap_value_t>{CAP_SETGID, CAP_SETUID}
     );
 
+    sleep(3);
```

Comment:
We need some discussion on this, this is added to wait for weston to actually listen X socket, so it can launch X server, then when pulseaudio's module-x11-bell calls XOpenDisplay(), it will trigger launch of X server, otherwise XOpenDisplay to fail. sleep() solution is not final (it just purpose of highlight issue for discussion).

Reply:
Resolved by wslgd-notify.